comp110_architecture.tex

```diff
@@ -5,41 +5,150 @@
 \usepackage[hidelinks]{hyperref}
 \usepackage[none]{hyphenat}
 \usepackage{setspace}
+\usepackage{graphicx}
+\graphicspath{ {G:/GitHubGames/comp110-architecture-essay/Pictures/} }
+	
 \doublespace
 
 % You may add additional packages here
 \usepackage{amsmath}
 
 % Please include a clear, concise, and descriptive title
-\title{Your Title Here}
+\title{Implementation of PCG: Three Examples But Which Is The Most Practical?}
```

Comment:
Avoid abbreviations in titles. Consider whether 'practical' is an appropriate basis upon which to make a recommendation.